app/templates/base_error.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    {% if title %}
+    <title>{{ title }}</title>
+    {% else %}
+    <title>Error</title>

```

Comment:
check out if there is a default-value filter for variable names. pretty sure there is — that way you can do 

<title>{{ title|'Error' }}</title>

or whatever the exact synatx is